common/static/js/vendor/pdfjs/viewer.html

```diff
@@ -110,7 +110,7 @@
 
             <a href="#" id="secondaryViewBookmark" class="secondaryToolbarButton bookmark visibleSmallView" title="Current view (copy or open in new window)" tabindex="55" data-l10n-id="bookmark">
               <span data-l10n-id="bookmark_label">Current View</span>
-            </a>
+          </a>
```

Comment:
@clrux Get this version of this file before your changes so you don't appear in the history.